readme.md

```diff
@@ -91,14 +105,13 @@ Here are some alternate ways of looping
 * `.times` (called on a number)
   * runs the loop a specific of times
```

Comment:
I feel like it'd be good to have one or both of the following here:

* an exercise (i.e. students try to come up with real-world use-cases for the various loop types)
* some context from us (instructors) about when these loops are used.